### PR TITLE
[feature] add links for revisions and commits

### PR DESCRIPTION
**Pull request by suevalov**
# Problem

Add missing `buildUrlFormat` feature to `cli`.

Closes https://github.com/paularmstrong/build-tracker/issues/65

# TODO

- [x] 🤓 Add & update tests (always try to _increase_ test coverage)
- [x] 🔬 Ensure CI is passing (`yarn lint:ci`, `yarn test`, `yarn tsc:ci`)
- [x] 📖 Update relevant documentation

### CODE REVIEW

**Comment by suevalov** [comment]
@paularmstrong could you please take a look?

File: src/cli/src/commands/__tests__/create-build.test.ts [hunk 1]
```diff
@@ -105,5 +110,51 @@ describe('create-build', () => {
...
+      jest.spyOn(Git, 'getDefaultBranch').mockReturnValue(Promise.resolve('master'));
+      jest.spyOn(Git, 'getBranch').mockReturnValue(Promise.resolve('tacobranch'));
+      jest.spyOn(Git, 'getParentRevision').mockReturnValue(Promise.resolve('1234567'));
+      jest.spyOn(Git, 'getCurrentRevision').mockReturnValue(Promise.resolve('abcdefg'));
+      jest
+        .spyOn(Git, 'getRevisionDetails')
+        .mockReturnValue(Promise.resolve({ timestamp: 1234567890, name: 'Jimmy', subject: 'tacos' }));
+
+      return Command.handler({ config: configWithFormatUrl, out: false, 'skip-dirty-check': true }).then(res => {
+        expect(res).toMatchObject({
```
Comment (paularmstrong) [inline src/cli/src/commands/__tests__/create-build.test.ts]: Can we cut this down to just the minimum:

```js
expect(res.meta.revision).toEqual({ url: 'https://github.com/paularmstrong/build-tracker/commit/abcdefg', value: 'abcdefg' });
```

File: src/cli/src/commands/create-build.ts [hunk 2]
```diff
@@ -41,6 +42,8 @@ export const builder = (yargs): Argv<Args> =>
       type: 'boolean'
     });
 
+type BuildMetaItem = string | { value: string; url: string };
```
Comment (paularmstrong) [inline src/cli/src/commands/create-build.ts]: Please import this:

```js
import { BuildMetaItem } from '@build-tracker/build';
```

File: src/cli/src/commands/create-build.ts [hunk 3]
```diff
@@ -70,12 +73,26 @@ export const handler = async (args: Args): Promise<{}> => {
   const { timestamp, name, subject } = await Git.getRevisionDetails(revision, config.cwd);
   const branch = await Git.getBranch(config.cwd);
 
+  const getRevisionItem = (): BuildMetaItem => {
```
Comment (paularmstrong) [inline src/cli/src/commands/create-build.ts]: Really minor stylistic change: it doesn't seem necessary to construct a function here.

```js
let revision = await Git.getCurrentRevision(config.cwd);
if (config.buildUrlFormat) {
  const toPath = pathToRegexp.compile(config.buildUrlFormat);
  const url = toPath({ revision });
  revision = {
    value: revision,
    url
  };
} 
```

**Comment by suevalov** [comment]
Thanks for the review! Fixed.

**Comment by suevalov** [comment]
hey @paularmstrong 👋 would you be able to take one more look, please?

**Comment by paularmstrong** [comment]
Published in v1.0.0-alpha.3

**Pull request merged**